docs/Creating-Your-Own-EmulationStation-Theme.md

```diff
@@ -1,166 +1,153 @@
-
-There will be a list of all links and resources at the end of the tutorial.
+    - Again, not essential, but it makes things easier. To be honest, as long as you know that `<tag>` opens a tag (more precisely an XML element) and `</tag>` closes a tag, then you should be fine.
 
----
+The (bogus) files you will need for this tutorial are listed [here](#references).
 
-## About Emulationstation Views
+### Primer on EmulationStation Views
 
-Emulationstation has 3 main sections, called Views:
-*[All images taken from the default Carbon theme]*
+EmulationStation has three main navigation sections, called _Views_:
 
-**System View**
-![Carbon Theme Detailed View](http://i.imgur.com/uTBYI4V.jpg)
-The System View is what you see when Emulationstation starts. It has a large white bar along the middle that houses a carousel that shows 3 System logos at a time.
+<figure markdown>
+  ![Carbon Theme System View](https://i.imgur.com/uTBYI4V.jpg)
+  <figcaption>Carbon Theme System View</figcaption>
+</figure>
+The System View is what you see when EmulationStation starts. It has a large white bar along the middle that houses a carousel that shows three System logos at a time.
 
-**Basic View**
-![Carbon Theme Detailed View](http://i.imgur.com/1C0WpSY.jpg)
+<figure markdown>
+  ![Carbon Theme Basic View](https://i.imgur.com/1C0WpSY.jpg)
+  <figcaption>Carbon Theme Basic View</figcaption>
+</figure>
 This is a simple game list page. You see this if you haven't scraped any metadata for that system.
 
-**Detailed View**
-![Carbon Theme Detailed View](http://i.imgur.com/pSHuOXK.jpg)
+<figure markdown>
+  ![Carbon Theme Detailed View](https://i.imgur.com/pSHuOXK.jpg)
+  <figcaption>Carbon Theme Detailed View</figcaption>
+</figure>
 The Detailed View is what you see if you *have* scraped metadata. Different themes can show different data, in different places or a different order, but every theme can only display these values (along with the Game List):
 
-- md_image
-- md_rating
-- md_releasedate
-- md_developer
-- md_publisher
-- md_genre
-- md_players
-- md_lastplayed
-- md_playcount
-- md_description
-
-**Video View**
-
-```Image Coming Soon```
-
-The Video View is what you see if your scraped metadata includes videos. The video view is an extension of the Detailed View.  It can display all of the values found in the Detailed View along with these additions:
+- `md_description`
+- `md_developer`
+- `md_genre`
+- `md_image`
+- `md_lastplayed`: Stored as ISO8601 date time format e.g., 2023-03-17T19:14:12+00:00
+- `md_playcount`
+- `md_players`
+- `md_publisher`
+- `md_rating`
+- `md_releasedate`: Stored as ISO8601 date format e.g., 2023-03-17
 
-- md_video
-- md_marquee
+Finally, the **Video View** is what you see if your scraped metadata includes videos. The video view is an extension of the Detailed View.  It can display all the values found in the Detailed View along with these additions:
```

Comment:
**Video View** is emphasized, but previously neither _Detailed_ nor _System_ were not.
I think they should all get the emphasis.

Reply:
I changed all View emphasis to italic, as there is a lot of bold already and I like the idea to use slanted text for anything on the screen/the user can interact with.